content/blog/2016/2016-09-28-jenkins-world-2016-wrap-up-hub.adoc

```diff
@@ -0,0 +1,149 @@
+---
+layout: post
+title: "Jenkins World 2016 Wrap-up - Open Source Hub"
```

Comment:
Really? It had a huge sign (with proper kerning!) saying "Open source hub" hanging over it, and I heard people referring to it as such… :)